docs/samples/Microsoft.ML.Samples.StaticPipe/Trainers.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.StaticPipe;
+using System;
+
+namespace Microsoft.ML.Samples.StaticPipe
+{
+    public static class Trainers
+    {
+        public static void Main()
```

Comment:
I think this "Main" method should be moved to a separate file. If we ever added new lines to this method (like if we wanted to run some other sample), then it will mess the line numbers up in:


```
        /// <![CDATA[
        ///  [!code-csharp[SDCA Regression](../../../docs/samples/Microsoft.ML.Samples.StaticPipe/Trainers.cs?range=5-8,12-14,25-79) "The SDCA regression example."]
        /// ]]></format>
````

Reply:
thinking more about it, we probably don;t need Main at all. We just care that this code compiles, afa running it, will probably do it only if we change things and want to validate the results. 

---
In reply to: [222419899](https://github.com/dotnet/machinelearning/pull/1105#discussion_r222419899) [](ancestors = 222419899)